taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -67,141 +65,161 @@ const SearchResults = () => {
-                        <InputLabel>Type</InputLabel>
-                        <Select
-                            native
-                            label="Type"
-                            value={nodeType}
-                            onChange={(e) => {
-                                setNodeType(e.target.value);
+                                {/* TODO: Add support for synonyms and stopwords */}
+                                <option value={'entry'}>Entry</option>
+                            </Select>
+                        </FormControl>
+                    </Stack>
+                    <Stack direction="row" alignItems="center" sx={{m: 2}}>
+                        <Typography>Main Language</Typography>
+                        <TextField
+                            onChange={(e) => { 
+                                setNewLanguageCode(e.target.value);
+                                const validateBool = ISO6391.validate(e.target.value);
+                                validateBool ? setisValidLC(true) : setisValidLC(false);
+                                validateBool ? setBtnDisabled(false) : setBtnDisabled(true);
                             }}
-                        >
-                            {/* TODO: Add support for synonyms and stopwords */}
-                            <option value={'entry'}>Entry</option>
-                        </Select>
-                    </FormControl>
-                </Stack>
-                <Stack direction="row" alignItems="center" sx={{m: 2}}>
-                    <Typography>Main Language</Typography>
-                    <TextField
-                        onChange={(e) => { 
-                            setNewLanguageCode(e.target.value);
-                            const validateBool = ISO6391.validate(e.target.value);
-                            validateBool ? setisValidLC(true) : setisValidLC(false);
-                            validateBool ? setBtnDisabled(false) : setBtnDisabled(true);
-                        }}
-                        label="Language Code"
-                        error={!isValidLC}
-                        sx={{width : 150, ml: 4.5}}
-                        size="small"
-                        variant="outlined"
-                    />
-                </Stack>
-                {
-                    nodeType === 'entry' &&
-                    <Stack direction="row" alignItems="center" sx={{mt: 2, ml: 2, mr: 2}}>
-                    <Typography>Node ID</Typography>
-                    <TextField
-                        onChange={(e) => { 
-                            setnewNode(e.target.value);
-                        }}
-                        label="Node ID"
-                        sx={{width : 150, ml: 11}}
-                        size="small"
-                        variant="outlined"
-                    />
+                            label="Language Code"
+                            error={!isValidLC}
```

Comment:
please improve the naming of `isValidLC`,  (kind of always better to avoid abbreviations, they get difficult to remember and understand very quickly)